sync/rpc/rpc.proto

```diff
@@ -1,67 +1,67 @@
 syntax="proto3";
 
 service SyncAgentService {
-    rpc RemoveFile(RemoveFileRequest) returns (Empty) {}
-    rpc RenameFile(RenameFileRequest) returns (Empty) {}
-    rpc LaunchReceiver(LaunchReceiverRequest) returns (LaunchReceiverReply) {}
-    rpc SendFile(SendFileRequest) returns (Empty) {}
-    rpc Coalesce(CoalesceRequest) returns (Empty) {}
-    rpc CreateBackup(CreateBackupRequest) returns (CreateBackupReply) {}
-    rpc RemoveBackup(RemoveBackupRequest) returns (Empty) {}
-    rpc RestoreBackup(RestoreBackupRequest) returns (Empty) {}
-    rpc RestoreBackupIncrementally(RestoreBackupIncrementallyRequest) returns (Empty) {}
+    rpc FileRemove(FileRemoveRequest) returns (Empty) {}
+    rpc FileRename(FileRenameRequest) returns (Empty) {}
+    rpc ReceiverLaunch(ReceiverLaunchRequest) returns (ReceiverLaunchReply) {}
```

Comment:
Move this function to the bottom or the top, so Filexxx can be grouped together.